internal/ethapi/api.go

```diff
@@ -961,7 +964,8 @@ func (diff *StateOverride) Apply(state *state.StateDB) error {
 		}
 		// Override account balance.
 		if account.Balance != nil {
-			state.SetBalance(addr, (*big.Int)(*account.Balance))
+			u256Balance, _ := uint256.FromBig((*big.Int)(*account.Balance))
```

Comment:
This could overflow, but I guess its not really important since a user can always set this field to uint256Max anyway

Reply:
Yeah. If a user specifies allocations such that the exceed 256 bits, then the warranty is off. Even if we catch this particular instance, as soon as he makes a tx adding one wei to an account at max, things will go south anyway.